modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/FormBuilder/FormBuilder.es.js

```diff
@@ -143,9 +143,9 @@ describe('Builder', () => {
 	});
 
 	afterEach(() => {
-		dom.exitDocument(addButton);
-		dom.exitDocument(basicInfo);
-		dom.exitDocument(translationManager);
+		addButton.remove();
```

Comment:
I tested running unit tests on dynamic-data-mapping modules  and it's running okay, without any unexpected error :)

Reply:
👏